### ISSUE TITLE
Add travis tests

### ISSUE THREAD

**Issue by tinmarino** [vimwiki/vimwiki#861]
To automatize testing on github.

Using the great test framework done by @ranebrown.

__Permits:__ faster accepting pull requests

**Comment by ranebrown** [comment]
Great idea! I've been meaning to do this for a while but haven't had the time.

**Comment by tinmarino** [comment]
Need vimwiki account credential (not only this repo) => @Nudin 
Follow [these](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci-using-github) 3 steps 3 minutes (can be done on smartphone).
You may need to make set your [membership public](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership) as advised in [this issue](https://github.com/travis-ci/travis-ci/issues/2519)

For example: here is the travis test in my local repo for the aborted PR I did : https://travis-ci.com/github/tinmarino/vimwiki/builds/165755728

**Comment by Nudin** [comment]
Is active now and I started a run on `dev` – that failed: [Build 167485097](https://travis-ci.com/github/vimwiki/vimwiki/builds/167485097)

**Comment by tinmarino** [comment]
Super Nudin.
I was responsible for the bad commit (Vint `!=` -> `!=#`): Fix: f54b94a38fd
```
autoload/vimwiki/base.vim:509:17: Use robust operators `!=#` or `!=?` instead of `!=` (see Google VimScript Style Guide (Matching))
```

Here the first good build [builds/167499425](https://travis-ci.com/github/vimwiki/vimwiki/builds/167499425).

Note the `nvim` complete removal and the `.travis cache` => you build compiled vim, mine used your cached compilation.

We now must keep the green check : [dev bracnh](https://github.com/vimwiki/vimwiki/commits/dev)

**Comment by tinmarino** [comment]
Travis tests are working.
I'm currently adding some tests and comments (in code) and will push them directly on dev branch (`Text:` and `Comment:` commit messages).
